packages/function-memoize-one/index.js

```diff
@@ -0,0 +1,44 @@
+module.exports = memoizeOne;
+
+function defaultEqual(arr1, arr2) {
+  if (arr1.length !== arr2.length) {
+    return false;
+  }
+
+  for (var i = 0; i < arr1.length; i++) {
+    if (arr1[i] !== arr2[i]) {
+      return false;
+    }
+  }
+
+  return true;
+}
+
+function memoizeOne(fn, isEqual) {
+  if (typeof fn !== 'function') {
+    throw new Error('fn should be a function');
+  }
+
+  if (isEqual !== undefined && typeof isEqual !== 'function') {
+    throw new Error('isEqual should be a function');
+  }
+
+  var wasCalled = false;
```

Comment:
I don't think we need the `wasCalled` variable since: 
  if never called before, lastArgs will be null
  if called before, lastArgs will always be an array